### DISCUSSION TITLE
Silent launch of 3.0.0?

### DISCUSSION THREAD

**Discussion by rhysperry111** [GNS3/gns3-gui#3680] [General]
Just wondering why the launch of 3.0.0 has been so silent - there hasn't been a blog post on gns3.com, nor has the main landing page been updated to list the new version.

Are you waiting for more people to try it before calling it mature enough to advertise on the main site?

**Comment by grossmj** [comment]
I intend to have a blog post soon. I just got delayed last weekend and with Xmas season ;)

> Are you waiting for more people to try it before calling it mature enough to advertise on the main site?

There is a bit of this too. v3 has some key changes and I don't want all users to switch to it now, especially the ones who won't read the blog post listing some warnings.

**Reply by rhysperry111** [reply]
> I intend to have a blog post soon. I just got delayed last weekend and with Xmas season ;)

No rush - enjoy your holiday! :christmas_island: :christmas_tree: :santa: :gift:

**Reply by grossmj** [reply]
Blog post now available: https://gns3.com/community/blog/gns3-3-0-released
